### DISCUSSION TITLE
Маршрутизация ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#40] [Announcements]
# Маршрутизация ⚡️ Node.js с примерами кода

Справочник web-разработчика с примерами кода - Node.js

https://nodejsdev.ru/doc/routing/

**Comment by prostohttp** [comment]
В блоке route() Лишняя ;
app.route('/api/users'); <- тут
    .post((req, res) => {...})
    .put((req, res) => {...})
    .delete((req, res) => {...});
